README.md

```diff
@@ -121,54 +125,68 @@ $ stack install
 
 ### Running the chat client
 
-To start the chat client, run `dog-food` (as in [eating your own dog food](https://en.wikipedia.org/wiki/Eating_your_own_dog_food)) from the terminal.
-
-By default, app data directory is created in the home directory (`~/.simplex`, or `%APPDATA%/simplex` on Windows), and SQLite database file `smp-chat.db` is initialized in it.
+To start the chat client, run `simplex-chat` from the terminal.
 
-The default SMP server is `smp1.simplex.im#pLdiGvm0jD1CMblnov6Edd/391OrYsShw+RgdfR0ChA=` (base-64 encoded string after server host is the transport key digest) - it is pre-configured in the app.
+By default, app data directory is created in the home directory (`~/.simplex`, or `%APPDATA%/simplex` on Windows), and two SQLite database files `simplex.chat.db` and `simplex.agent.db` - for chat client and agent respectively - are initialized in it.
 
-To specify a different file path for the chat database use `-d` command line option:
+To specify a different file path prefix for the database files use `-d` command line option:
 
 ```shell
-$ dog-food -d my-chat.db
+$ simplex-chat -d alice
 ```
 
+Running above, for example, would create `alice.chat.db` and `alice.agent.db` database files in current directory.
+
+Default SMP servers are `smp2.simplex.im#z5W2QLQ1Br3Yd6CoWg7bIq1bHdwK7Y8bEiEXBs/WfAg=` (UK) and `smp3.simplex.im#nxc7HnrnM8dOKgkMp008ub/9o9LXJlxlMrMpR+mfMQw=` (California) - they are pre-configured in the app. Base-64 encoded string after server host is the transport key digest.
```

Comment:
are hosted on Linode ... London, UK / Freemont, CA. No need to put the addresses here, you can link to the source code.